otx/algorithms/segmentation/tasks/inference.py

```diff
@@ -212,11 +212,10 @@ def _init_recipe(self):
         logger.info(f"initialized recipe = {recipe}")
 
     def _update_stage_module(self, stage_module: str):
-        if self._train_type == TrainType.SEMISUPERVISED:
-            if stage_module == "SegTrainer":
-                return "SemiSegTrainer"
-            if stage_module == "SegInferrer":
-                return "SemiSegInferrer"
+        module_prefix = {TrainType.SEMISUPERVISED: "SemiSL", TrainType.INCREMENTAL: "Incr"}
```

Comment:
As far as I know, there is `selfsl` for segmentation, but does it just use `SegTrainer`?